src/Presentation/Detection/Creating/Types/TCreateDetection.cs

```diff
@@ -0,0 +1,18 @@
+namespace Presentation.Detection.Common.Types
```

Comment:
Should we also update the namespace to match the new folder structure.

Reply:
Done